sqltoolsservice.sln

```diff
@@ -50,6 +50,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.SqlTools.Credenti
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.SqlTools.Hosting", "src\Microsoft.SqlTools.Hosting\Microsoft.SqlTools.Hosting.csproj", "{AAE1F8D1-F7AB-4ABE-A55B-D423393AB352}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.SqlTools.SqlCore", "src\Microsoft.SqlTools.SqlCore\Microsoft.SqlTools.SqlCore.csproj", "{C175C3D4-1AE1-4B62-85EF-8298D37F2BAC}"
```

Comment:
Do make sure this project DLL is added to the signing list in [build.yml](https://github.com/microsoft/sqltoolsservice/blob/main/azure-pipelines/build.yml#L166) and also in the ADS repo: https://github.com/microsoft/azuredatastudio/blob/606d8086418046b0a13afbe49c2fd86c502f2288/build/azure-pipelines/win32/sql-product-build-win32.yml#L187
if it should be used by itself by any product.